omero/training-server/playbook.yml

```diff
@@ -319,7 +319,7 @@
     # You may wish to change this variable,
     # or override it by defining the private variable os_system_users_password_override.
     os_system_users_password: "{{ os_system_users_password_override | default('$6$leKi5B1PgSvdA/ec$xbU3CnoSFnYdeZjEjKK5TH8SGATsW746uopssff4edpgyu.cWXGo9A.oK6wH9kIkxLCCNcORGZnnroZPMqGzN/') }}"
-    apache_docker_release: "{{ apache_docker_release_override | default('0.3.0') }}"
+    apache_docker_release: "{{ apache_docker_release_override | default('0.6.0') }}"
```

Comment:
You could delete `apache_docker_release_override` too, so it's just `apache_docker_release: '0.6.0'`?

Reply:
But note that this is in line with the overall override system for all the variables ?
Is it advisable to have some of them with the override variable, some not ?
Definitely, some of the variables here must be overriden.